### PR TITLE
CLDR-16263 update ISO-4217 data since HRK is obsolete

### PR DESCRIPTION
**Pull request by srl295**
will do a proper import in CLDR-16317

CLDR-16263

- [ ] This PR completes the ticket.

ALLOW_MANY_COMMITS=true

### CODE REVIEW

**Review by btangmu — APPROVED** [review]

**Comment by srl295** [comment]
patch was too 'hot'.  Retrying with full process per https://cldr.unicode.org/development/updating-codes/update-currency-codes - needed correct update to ISO4217.txt

**Comment by srl295** [comment]
and it broke due to TÜRKİYE

File: tools/cldr-code/src/main/resources/org/unicode/cldr/util/data/dl_iso_table_a1.xml [hunk 1]
```diff
@@ -1708,7 +1708,7 @@
 			<CcyMnrUnts>3</CcyMnrUnts>
 		</CcyNtry>
 		<CcyNtry>
-			<CtryNm>TURKEY</CtryNm>
+			<CtryNm>TÜRKİYE</CtryNm>
```
Comment (srl295) [inline tools/cldr-code/src/main/resources/org/unicode/cldr/util/data/dl_iso_table_a1.xml:1711]: OK seriously ISO-4217 MA? `TÜRKİYE` with a dotted capital I?

File: tools/cldr-code/src/main/resources/org/unicode/cldr/util/data/dl_iso_table_a3.xml [hunk 2]
```diff
@@ -1,1136 +1,1143 @@
...
+			<CcyNbr>191</CcyNbr>
+			<WthdrwlDt>1995-01</WthdrwlDt>
+		</HstrcCcyNtry>
+		<HstrcCcyNtry>
+			<CtryNm>CROATIA</CtryNm>
+			<CcyNm>Croatian Kuna</CcyNm>
+			<Ccy>HRK</Ccy>
+			<CcyNbr>191</CcyNbr>
+			<WthdrwlDt>2015-06</WthdrwlDt>
+		</HstrcCcyNtry>
```
Comment (macchiati) [inline tools/cldr-code/src/main/resources/org/unicode/cldr/util/data/dl_iso_table_a3.xml:255]: Had to "hide whitespace changes" to review this effectively.

**Review by macchiati — APPROVED** [review]

File: tools/cldr-code/src/main/resources/org/unicode/cldr/util/data/dl_iso_table_a3.xml (hunk 2 above)
Comment (srl295) [reply tools/cldr-code/src/main/resources/org/unicode/cldr/util/data/dl_iso_table_a3.xml:255]: That's odd. I don't see extraneous whitespace.

**Pull request merged**